shell/platform/tizen/tizen_renderer_evas_gl.cc

```diff
@@ -671,19 +671,24 @@ bool TizenRendererEvasGL::SetupEvasWindow() {
     initial_geometry_.h = height;
   }
 
-  elm_win_alpha_set(evas_window_, EINA_FALSE);
   evas_object_move(evas_window_, initial_geometry_.x, initial_geometry_.y);
   evas_object_resize(evas_window_, initial_geometry_.w, initial_geometry_.h);
   evas_object_raise(evas_window_);
 
   elm_win_indicator_mode_set(evas_window_, ELM_WIN_INDICATOR_SHOW);
   elm_win_indicator_opacity_set(evas_window_, ELM_WIN_INDICATOR_OPAQUE);
 
-  Evas_Object* bg = elm_bg_add(evas_window_);
-  evas_object_color_set(bg, 0x00, 0x00, 0x00, 0x00);
+  if (transparent_) {
+    elm_win_alpha_set(evas_window_, EINA_TRUE);
```

Comment:
Will setting `transparent` as true affect performance? https://github.com/flutter-tizen/engine/issues/222

Reply:
Yes, as far as I know, it can't get EFL optimize rendering support if we specify alpha property as true.